x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -375,57 +375,49 @@ export const EndpointList = () => {
   ]);
 
   return (
-    <ManagementPageView
-      viewType="list"
-      data-test-subj="endpointPage"
-      headerLeft={
-        <>
-          <EuiFlexGroup alignItems="center" responsive={false}>
-            <EuiFlexItem grow={false}>
-              <EuiTitle size="l">
-                <h1 data-test-subj="pageViewHeaderLeftTitle">
-                  <FormattedMessage
-                    id="xpack.securitySolution.endpoint.list.pageTitle"
-                    defaultMessage="Endpoints"
-                  />
-                </h1>
-              </EuiTitle>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <EuiBetaBadge
-                label={i18n.translate('xpack.securitySolution.endpoint.list.beta', {
-                  defaultMessage: 'Beta',
-                })}
-              />
-            </EuiFlexItem>
-          </EuiFlexGroup>
-          <EuiSpacer size="s" />
-          <EuiText size="s" color="subdued">
-            <p>
+    <WrapperPage noTimeline data-test-subj="endpointPage">
```

Comment:
Regarding the this test failure: https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/67945/testReport/junit/X-Pack%20Jest%20Tests/x-pack_plugins_security_solution_public_management_pages/when_in_the_Admistration_tab_should_display_the_Management_view_when_Ingest_is_ON/

it doesn't look like `<WrapperPage>` renders in the html with the `data-test-subj`, putting it on the `<HeaderPage>` below fixes it.

This nuance could be another good reason to wrap this pattern as @paul-tavares suggests.  What are your thoughts?

Reply:
I accidentally didn't copy the full solution I had on other branch. I completed it now so tests should succeed. As for the shared component - as indicated in the other resolved comment it's partially done now (for list pages).